src/training/zero_shot.py

```diff
@@ -11,7 +11,8 @@
 
 
 def zero_shot_classifier(model, classnames, templates, args):
-    tokenize = HFTokenizer(args.hf_tokenizer_name) if args.hf_tokenizer_name else tokenize
+    if hasattr(model, "text") and hasattr(model.text, "tokenizer"):
```

Comment:
can we also have it at model.tokenizer that way we can let users always use model.tokenizer for normal tokenize or hf (for pretrained models) ?

Reply:
and this should be documented in readme